### ISSUE TITLE
address build warnings in ncurses example

### ISSUE THREAD

**Issue by 2bndy5** [TMRh20/RF24Gateway#35]
just a self-reminder.
```
/home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/examples/ncurses/RF24Gateway_ncurses.cpp: In function ‘void drawTopology()’:
/home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/examples/ncurses/RF24Gateway_ncurses.cpp:375:25: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
  375 |                     int y = 0;
      |                         ^
/home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/examples/ncurses/RF24Gateway_ncurses.cpp:397:25: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
  397 |                     int y = 0;
      |                         ^
/home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/examples/ncurses/RF24Gateway_ncurses.cpp:419:25: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
  419 |                     int y = 0;
      |                         ^
/home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/examples/ncurses/RF24Gateway_ncurses.cpp: In function ‘void drawRF24Pad()’:
/home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/examples/ncurses/RF24Gateway_ncurses.cpp:667:42: warning: format ‘%u’ expects argument of type ‘unsigned int’, but argument 3 has type ‘size_t’ {aka ‘long unsigned int’} [-Wformat=]
  667 |     wprintw(rf24Pad, "RX Packets(user): %u\n", networkPacketsRX);
      |                                         ~^     ~~~~~~~~~~~~~~~~
      |                                          |     |
      |                                          |     size_t {aka long unsigned int}
      |                                          unsigned int
      |                                         %lu
/home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/examples/ncurses/RF24Gateway_ncurses.cpp: In function ‘void drawConnPad()’:
/home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/examples/ncurses/RF24Gateway_ncurses.cpp:714:44: warning: format ‘%d’ expects argument of type ‘int’, but argument 5 has type ‘size_t’ {aka ‘long unsigned int’} [-Wformat=]
  714 |             mvwprintw(connPad, ctr++, 0, "%d %s\n", lCtr++, line.c_str());
      |                                           ~^        ~~~~~~
      |                                            |            |
      |                                            int          size_t {aka long unsigned int}
      |                                           %ld
```

**Comment by 2bndy5** [comment]
Locally, I'm also seeing
```
/home/brendan/repos/RF24Gateway/examples/ncursesInt/RF24Gateway_ncursesInt.cpp:190:28: warning: ‘header.RF24NetworkHeader::type’ may be used uninitialized [-Wmaybe-uninitialized]
  190 |                 if (header.type == 1) // header.type is uninitialized
      |                     ~~~~~~~^~~~
/home/brendan/repos/RF24Gateway/examples/ncursesInt/RF24Gateway_ncursesInt.cpp:184:35: note: ‘header’ declared here
  184 |                 RF24NetworkHeader header;
      |                                   ^~~~~~
```
And rightfully so https://github.com/nRF24/RF24Gateway/blob/635e520e8c6746754d5799853aa551c30799a6dc/examples/ncursesInt/RF24Gateway_ncursesInt.cpp#L179-L208

Partly, I think this uninitialized `header.type` is a actually an oversight in `RF24NetworkHeader()` (empty c'tor). Although, the usage here makes a fresh instance of a header and emits a timestamp if the `header.type` is uninitialized. Logically speaking, this timestamp would be sent every time the header is constructed. So, I think the solution is just send the timestamp unconditionally (to master since `header.from_node` is supposedly also uninitialized).

**Comment by 2bndy5** [comment]
> Locally, I'm also seeing ...

This was fixed in #36 . We now build IRQ examples in CI, so the warnings are shown there too.
